### ISSUE TITLE
parametrize plugins usage

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#300]
To finalize with opencv we have to add additionally images plugin for thins version.
So far it is enabled in the simple and opencv version. Let's find a way to parametrize plugins and use it via env vars if any and remove from default one

**Comment by vdelendik** [comment]
done, we have to test both images using basic acceptance for ios and android acceptance:
appium:2.0.9
appium:2.0.9-opencv5.6.0-beta9

1st one is without images plugins and 2nd one has images plugin + opencv installed

**Comment by vdelendik** [comment]
unable to start opencv version using images plugin:
```
docker logs -f device-iPhone_X_v15-9f53aa9bd1638255f48ca6beac4285cae4f43eef-appium
/opt/entrypoint/entrypoint.sh: line 8: images: command not found
Device is not available!
Exiting without restarting...
```

**Comment by vdelendik** [comment]
fixed in better way in `2.0.9-opencv5.6.0-beta11`

**Comment by dhreben** [comment]
Will be verify on prod
